### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by artemkazakov947**
Please, check it.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, int):
```
Comment (IvanRamyk) [inline app/main.py:21]:
```suggestion
        if isinstance(other, (int, float)):
```
Distance class should support both numerical types

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by artemkazakov947 — COMMENTED** [review]
float was added to isinstat()

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,63 @@
+from __future__ import division
```
Comment (IvanRamyk) [inline app/main.py:1]: Please, don't leave unused imports in the code

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (artemkazakov947) [reply app/main.py]: Ooops

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Review by IvanRamyk — APPROVED** [review]

**Pull request closed without merging**